pkg/api/v2/reporter/reporter.go

```diff
@@ -53,6 +56,17 @@ func (e ResourceReports) Merge(resErrs ResourceReports) {
 				}
 			}
 
+			// Merge messages lists
+			allMessages := make(map[string]bool)
```

Comment:
nit; style. let's use a map[string]struct{} whenever we don't need the value and are using a map to implement a set

Reply:
Like last time, got it!